samples/AspNetCoreApiSample/Utils/PropertiesChangeTracker.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Withywoods.AspNetCoreApiSample.Utils
+{
+    /// <summary>
+    /// A tracker that allows to keep track of any changed properties
+    /// </summary>
+    public class PropertiesChangedTracker : INotifyPropertyChanged
```

Comment:
Why do we need this file?
I don't see it used in code referencing TaskPatchDto.
(BTW Utils is a too generic word)
It would be best to have the code as much simple as possible.

Reply:
This class allows to track which properties were changed since the instanciation or reinitialization (with AcceptChanges method).
This is useful to differentiate for example null properties because the value was never modified (so in a PATCH behavior, this value will not be modified in repository), from null value which was explicitly set (the value has to be erased in the repository).
There are some libraries which allows this tracking, but they are not free (PostSharp) or need a configuration that seems to be not compliant with hexagonal architecture (eg: TrackerDog).

Reply:
Interesting!
Would it be possible to add this test scenario in the integration tests? (do a patch with at least one null value then check the values returned by the get)